### PR TITLE
WIP: Add community info update

### PR DESCRIPTION
**Pull request by jkilpatr**

### CODE REVIEW

File: docs/updates/infromational-dec-26-2021.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+The information in this document is accurate, to the best of our knowledge as of December 26th 2021
+
+## Delegation Bug, diagnosis and fix
+
+### Issue summary
+
+Quickly after the launch of `gravity-bridge-1` some users reported [issues](https://github.com/Gravity-Bridge/Gravity-Bridge/issues/2) withdrawing their delegated tokens.
+
+This issue has been root caused and resolved in [this patch](https://github.com/Gravity-Bridge/Gravity-Bridge/commit/146b11fcfd8d6c4dcb541507d634839f37e0cce8). The root cause is that during the initialization process for the Gravity Bridge module staking hooks where being set on only one of two copies of the StakingKeeper.
```
Comment (ChristianBorst) [inline docs/updates/infromational-dec-26-2021.md:15]: where not being set -> were not being set

File: docs/updates/infromational-dec-26-2021.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+StakingHooks are pieces of code that CosmosSDK calls when events related to staking occur. Specifically for the Gravity module Slashing.
+
+Due to an oversight in the Gravity Bridge module staking hooks where being set on one copy of the StakingKeeper but not on the copy being used by the Gravity Bridge module when performing slashing. This results in several hooks not running when a validator is slashed by the Gravity Bridge module.
```
Comment (ChristianBorst) [inline docs/updates/infromational-dec-26-2021.md:19]: **where** being set -> **were* being set

File: docs/updates/infromational-dec-26-2021.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+In this specific case the hooks take care of recording that a slashing event has occurred in the StakingKeeper, but the actual modification of the users stake does not depend on the hook.
```
Comment (ChristianBorst) [inline docs/updates/infromational-dec-26-2021.md:21]: of the users stake -> of the user**'s** stake

File: docs/updates/infromational-dec-26-2021.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+} total affected stake 1803249988137
+
+```
+
+As you can see all affected delegations are self-delegations to genesis validators. Each holding `181,818 graviton` with slight
+variations based on withdraws and redelegation before their slashing and exactly how many times they have been slashed by the Gravity module.
+
+### Recommended Action
+
+The latest release version of Gravity Bridge `v1.1.0` and greater correctly set the hooks in the Gravity module `app.go`. It is important that the community upgrade to `v1.1.0` or later release or any validator that is slashed by the Gravity Bridge module will be unable to withdraw delegation rewards.
```
Comment (ChristianBorst) [inline docs/updates/infromational-dec-26-2021.md:67]: It is important that the community upgrade to `v1.1.0` or later release or any validator that is slashed by the Gravity Bridge module will be unable to withdraw delegation rewards.
->
It is important that the community upgrades to release `v1.1.0` or later as any validator slashed by the Gravity Bridge module will be unable to withdraw delegation rewards. Additionally, delegators to such a validator will be unable to withdraw the funds they have delegated to that validator, along with any rewards they have earned from that delegation.

File: docs/updates/infromational-dec-26-2021.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+Although unbonding of the delegated tokens will function correctly all delegators to that validator, which includes the validator themselves, will be unable to withdraw rewards after the slashing occurs.
```
Comment (ChristianBorst) [inline docs/updates/infromational-dec-26-2021.md:69]: I don't believe this is true, Undelegating in staking module will trigger the `BeforeDelegationSharesModified()` hook in distribution, which calls the problematic `withdrawDelegationRewards` function and panics on calculating their rewards.

Comment (ChristianBorst) [reply docs/updates/infromational-dec-26-2021.md:69]: This is the same story for Redelegation, eventually the panic occurs on that distribution sanity check.

File: docs/updates/infromational-dec-26-2021.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+### Resolution for affected individuals
+
+The most correct way to resolve this issue is to insert the correct slashing events into the history during a chain upgrade. This would allow the affected validators to withdraw their rewards with no disruption.
+
+Sadly this option has significant downsides. Since it can only be done during chain upgrade the upgrade to `v1.1.0` can either be delayed. Risking more validators encountering the issue, or the affected users may have to wait until the following upgrade to be able to withdraw their rewards.
```
Comment (ChristianBorst) [inline docs/updates/infromational-dec-26-2021.md:75]: can either be delayed. Risking more validators encountering the issue, or ...
->
can either be delayed, risking more validators encountering the issue, or ...

File: docs/updates/infromational-dec-26-2021.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+
+Furthermore given the compounding effect of withdrawing and re-delegating simply allowing affected individuals to withdraw rewards does not compensate for opportunity cost.
+
+The community may choose to use a Gravity Bridge [Airdrop Proposal](https://github.com/Gravity-Bridge/Gravity-Bridge/blob/main/module/proto/gravity/v1/types.proto#L67) to send users who have experienced this issue a flat amount that can be computed to include opportunity cost.
+
+### Recommendation for affected individuals
+
+Regardless of how the community choses to resolve this issue the fastest way to get back on track staking for an affected individual is to begin unbonding and re-delegate their tokens to a new validator, unaffected by the bug. Once the unbonding period ends.
```
Comment (ChristianBorst) [inline docs/updates/infromational-dec-26-2021.md:83]: Again I'm fairly certain that an affected validator will not be able to move their delegation around or undelegate based on the staking and distribution code I have looked at. Were you able to test this?

File: docs/updates/infromational-dec-26-2021.md (hunk 5 above)
Comment (jkilpatr) [reply docs/updates/infromational-dec-26-2021.md:69]: I was worried about that, but never got around to testing it using my new integration test on Sunday. Thanks for flagging. It does elevate this bug, if anyone else is effected an airdrop proposal won't be a reasonable resolution.

File: docs/updates/infromational-dec-26-2021.md (hunk 7 above)
Comment (jkilpatr) [reply docs/updates/infromational-dec-26-2021.md:83]: testing it locally now. Will get back to you.

Comment (jkilpatr) [reply docs/updates/infromational-dec-26-2021.md:83]: haven't really been able to test conclusively. My laptop isn't really fast enough to get good data here. Going to go ahead with the assumption that unbonding does not work.

**Pull request merged**